src/dali/plugin/pair_converter/csv/kraken.py

```diff
@@ -351,8 +363,17 @@ def _google_file_to_bytes(self, file_name: str) -> Optional[bytes]:
                             error[_MESSAGE],
                         )
                         raise RP2RuntimeError("Google Drive key invalid")
+                    if error[_REASON] == _INVALID_VALUE:
```

Comment:
When was this error coming up? If we have an invalid value, we should filter that out before we send a request.

Reply:
I disagree, its a valid return value from the API.  It's been a while now, so I can't really say but it's probably as it says "an invalid value" was sent to the API.

Reply:
It's fine enough as a backstop, but it would be helpful to users if we could validate the parameters and give them clearer feedback, so I was curious what was causing it.

Reply:
A little tough now since its been a few weeks, but I think its related to an error in my query when fixing the `spot_price` issue. It shouldn't happen if things are coded properly but while I was working on it, this would help provide insight.

Reply:
all right, let's just leave it and if it comes up then we will have a trace to look at.